src/components/Issues/IssuesListView/IssuesListHeader.tsx

```diff
@@ -0,0 +1,49 @@
+export const issuesTableColumnClasses = {
+  issue: 'pf-m-width-20 wrap-column',
+  scope: 'pf-m-width-10',
+  severity: 'pf-m-width-10',
+  status: 'pf-m-width-10',
+  createdOn: 'pf-m-width-15',
+  reason: 'pf-m-width-20',
+  usefulLinks: 'pf-m-width-10',
+  kebab: 'pf-m-width-5 issues-list-view__actions',
+};
+
+const IssuesListHeader = () => {
```

Comment:
curious question: shouldn't we add sorting functionality? 🤔 

I'm asking that because I noticed that some columns have the up/down arrows in the UX doc:

![Image](https://github.com/user-attachments/assets/75d0f0b9-8534-495c-b748-5ee244c62f99)